### ISSUE TITLE
New Release

### ISSUE THREAD

**Issue by bdevcich** [hpc/mpifileutils#558]
Any plans for a new release? It's been quite a while since v0.11.1 was cut.

**Comment by adammoody** [comment]
Yes, good point.  I'll work on this over the next month or so.  There are a few things I'd like to see if I can work in.

**Comment by bdevcich** [comment]
Any traction on this?

**Comment by bdevcich** [comment]
https://github.com/hpc/mpifileutils/pull/623
